components/Footer.tsx

```diff
@@ -43,12 +43,11 @@ const Footer: FC<FooterProps> = ({
   footerNav1,
   footerNav2,
 }) => {
-  const commonLocale = useCommonLocale()
-
+  const { t } = useTranslation('common')
```

Comment:
Components shouldn't be responsible for their own translation; all the text should get passed to it's props from the page level. We may want to revisit this after, as it would be significant change to all the components edited in this PR.